### PR TITLE
[GH-1325] Add the previously missing fields.

### PR DESCRIPTION
**Pull request by rexwangcc**
### Purpose

- https://broadinstitute.atlassian.net/browse/GH-1325

### Changes

- No changes.

#### What does this PR do?

Please also refer to the Slack discussion [here](https://broadinstitute.slack.com/archives/CFA6Q2QBU/p1621872517090900). 

Basically, we need to make sure we also include the optional `Control-` files and field when those fields appear in the JMS message, with a naming transformation to make the Arrays WDL happy:
```
File? control_sample_vcf_file
File? control_sample_vcf_index_file
File? control_sample_intervals_file
String? control_sample_name
```

According to Geroge, these files are static and not changing between runs, but in order to make AoU SA see them during execution, we need to put them in our custom buckets anyways. To avoid having to also transform the values of the fields, and given the relatively small size (X MiB per file) of these files, we can just copy them into the bucket for **each** sample, instead of copying them to a well-known place in advance. This behavior is consistent with how we have been treating CHIP files such as `beadPoolManifestPath` and `genderClusterFilePath`.

### CODE REVIEW

**Review by tbl3rd — APPROVED** [review]
Looks good thanks.  Tests still running though.

File: test/data/bogus-control-intervals.interval_list [hunk 1]
```diff
@@ -0,0 +1 @@
+bogus-control-intervals.interval_list
```
Comment (rexwangcc) [inline test/data/bogus-control-intervals.interval_list:1]:
```suggestion
bogus-control-intervals.interval_list

```

File: test/data/bogus-control-vcf-index.vcf.gz.tbi [hunk 2]
```diff
@@ -0,0 +1 @@
+bogus-control-vcf-index.vcf.gz.tbi
```
Comment (rexwangcc) [inline test/data/bogus-control-vcf-index.vcf.gz.tbi:1]:
```suggestion
bogus-control-vcf-index.vcf.gz.tbi

```

File: test/data/bogus-control-vcf.vcf.gz [hunk 3]
```diff
@@ -0,0 +1 @@
+bogus-control-vcf.vcf.gz
```
Comment (rexwangcc) [inline test/data/bogus-control-vcf.vcf.gz:1]:
```suggestion
bogus-control-vcf.vcf.gz

```

**Comment by rexwangcc** [comment]
We decided to leave the test untouched as it requires a fair amount of engineering effort and time to find and update to a controlled sample. For this specific issue, we can verify it works easily by looking at the PTC-generated JSON as well as the WFL-started workflow.

**Pull request merged**